common/reviews/api/package-deps-hash.api.md

```diff
@@ -7,6 +7,9 @@
 // @public
 export function getPackageDeps(packagePath?: string, excludedPaths?: string[]): IPackageDeps;
 
+// @public
+export function gitHashFiles(filesToHash: string[], packagePath: string): Map<string, string>;
```

Comment:
Our convention is for functions to start with an English verb.  "git" isn't a very professional sounding verb heheh.

`getGitHashForFiles`?  Or merely `getHashForFiles`?